openfga/v1/authzmodel.proto

```diff
@@ -126,3 +131,25 @@ message TupleToUserset {
   ObjectRelation tupleset = 1;
   ObjectRelation computed_userset = 2;
 }
+
+message Condition {
+  // A unique name for the condition
+  // TODO(@jpadilla): Add validation rules
```

Comment:
In this PR?

Reply:
let's follow up on a separate pull request.

Reply:
Validation rules are part of the API spec though. Splitting that to another PR is just going to create more noise and context switching. Please include it here in the scope of this PR.